packages/scripts/scripts/lint-md-docs.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+const { sync: spawn } = require( 'cross-spawn' );
+const { sync: resolveBin } = require( 'resolve-bin' );
+
+/**
+ * Internal dependencies
+ */
+const {
+	fromConfigRoot,
+	getArgsFromCLI,
+	hasArgInCLI,
+	hasFileArgInCLI,
+	hasProjectFile,
+} = require( '../utils' );
+
+const args = getArgsFromCLI();
+
+const defaultFilesArgs = hasFileArgInCLI() ? [] : [ '**/*.md' ];
+
+// See: https://github.com/igorshubovych/markdownlint-cli#configuration
+// Check if specified on command-line or project file for config
+const hasLintConfig = hasArgInCLI( '-c' ) || hasArgInCLI( '--config' ) ||
+	hasProjectFile( '.markdownlint.json' ) ||
+	hasProjectFile( '.markdownlint.yaml' ) ||
+	hasProjectFile( '.markdownlint.yml' );
+
+// When a configuration is not provided by the project, use from the default
+// provided with the scripts module.
+const defaultConfigArgs = ! hasLintConfig ?
+	[ '--config', fromConfigRoot( '.markdownlint.json' ) ] :
+	[];
+
+// See: https://github.com/igorshubovych/markdownlint-cli#ignoring-files
+// Check if ignore specified on command-line or project file
+const hasIgnoredFiles = hasArgInCLI( '--ignore' ) || hasArgInCLI( '-i' ) ||
+	hasProjectFile( '.markdownlintignore' );
+
+// Default ignore [ build, node_modules ] directories
+// TODO: Once https://github.com/igorshubovych/markdownlint-cli/issues/46 is in
```

Comment:
Great idea to add a comment with future improvements possible.

Reply:
Thanks, I'll forget to follow up without it

Reply:
I just stumbled on this one, looks like the related issue is now fixed. So should we follow-up?

Reply:
👍 Updated in #32633